UPGRADE-6.7.md

```diff
@@ -1,20 +1,86 @@
-# 6.7.0.0
-## Introduced in 6.6.9.0
-## Administration removed associations
-* Removed `calculationRule` association in `shippingMethodCriteria()` in `sw-settings-shipping-detail`.
-* Removed `conditions` association in `ruleFilterCriteria()` and `shippingRuleFilterCriteria()` in `sw-settings-shipping-price-matrix`
-## Parameter names of some `\Shopware\Core\Framework\Migration\MigrationStep` changed
-* Parameter name `column` of `\Shopware\Core\Framework\Migration\MigrationStep::dropColumnIfExists` changed to `columnName` 
-* Parameter name `column` of `\Shopware\Core\Framework\Migration\MigrationStep::dropForeignKeyIfExists` changed to `foreignKeyName` 
-* Parameter name `index` of `\Shopware\Core\Framework\Migration\MigrationStep::dropIndexIfExists` changed to `indexName`
-## Removal of deprecated product review loading logic in Storefront
-* The service `\Shopware\Storefront\Page\Product\Review\ProductReviewLoader` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\AbstractProductReviewLoader` instead.
-* The event `\Shopware\Storefront\Page\Product\Review\ProductReviewsLoadedEvent` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\Event\ProductReviewsLoadedEvent` instead.
-* The hook `\Shopware\Storefront\Page\Product\Review\ProductReviewsWidgetLoadedHook` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewsWidgetLoadedHook` instead.
-* The struct `\Shopware\Storefront\Page\Product\Review\ReviewLoaderResult` was removed. Use `\Shopware\Core\Content\Product\SalesChannel\Review\ProductReviewResult` instead.
-## Native types for PHP class properties
-All PHP class properties now have a native type.
-If you have extended classes with properties, which didn't have a native type before, make sure you now add them as well.
+# 6.7.0.0 Upgrade Guide
+**NOTE:** All the breaking changes described here can be already opted in by activating the `v6.7.0.0` [feature flag](https://developer.shopware.com/docs/resources/references/adr/2022-01-20-feature-flags-for-major-versions.html#activating-the-flag) on previous versions.
+
+# Notable Changes
+
+# Webpack to vite migration for the administration
+We are switching the build system for our administration from webpack to vite. 
+This means that when your plugins depends on a custom `webpack.config.js` file, you'll need to migrate it to a `vite.config.js` file.
+**TBD how does that look like? Is there a docs page for that?**
```

Comment:
@shopware/product-ct-admin can you double check the admin related changes here? 

I added some questions were I'll need more input, would be great if you can provide that, will tag you in the other sections as well